### DISCUSSION TITLE
Problem with Construction timelapse

### DISCUSSION THREAD

**Discussion by iainballantyne-pixel** [gopro/labs#212] [General]
Hi, I bought a GoPro 11 to use as a time lapse for our new house extension. I'm new to all this, so please bear with me. I've downloaded the QR firmware which allows the GoPro to scan the QR codes on here. I've then used the construction time lapse ready made QR code on here (after using the QR code to set the time). The QR code scans and I get a message stating the time the images will start and the camera powers off. If I leave the camera, at the correct time, it seems to power up and appears to take an image, it then powers down again and repeatedly does this for the set duration.

However, at the end of the duration, if I look on the media section (swipe up on the screen) there are no images/videos. If I take the SD card out and look at it, there are no files at all on the memory card.

Construction starts on Monday, any help would be appreciated, I've been struggling with this for 2 weeks now! 

Thanks in advance.

Iain

**Comment by iainballantyne-pixel** [comment]
Actually, having just paid more attention to it, at the time it's due to start, it powers up and starts a countdown, then powers off immediately before the countdown completes. it then switches on again briefly and then stays off for the rest of the set duration.

**Comment by iainballantyne-pixel** [comment]
Last one, I also have a GoPro Max and have done the exact same thing with it and it works perfectly, except it's no good because you only seem to get a 360 image which isn't what I'm looking for.

Any help appreciated.

**Comment by dnewman-gpsw** [comment]
There is a bug in the last HERO11 Labs firmware that will be addressed in the next release.  In the meantime the previous firmware supports this command https://github.com/gopro/labs/blob/master/docs/firmware/bin/LABS_HERO11_01_20_70.zip

MAX can also do this, you just need to select the single lens mode if you don't want 360 images.

**Comment by iainballantyne-pixel** [comment]
Thank you so much, it has worked!

**Comment by don-jaime** [comment]
Just noting that this problem seems to also be introduced on the MAX 360 on its latest labs firmware, `LABS_MAX_02_00_75.zip`. It's not happening for me on `LABS_MAX_02_00_71.zip`, so rolling back to that is a fine workaround.

**Comment by filipeg165** [comment]
Hello, I'm having this problem on a Hero 10 as well, using firmware 1.47, is there any other means to fix this on my camera?

Right before the countdown ends, the camera shuts off and never back on again.

**Reply by dnewman-gpsw** [reply]
FYI, HERO10 is now on v1.50.70.

This was a bug in H11, addressed months ago.  I'm not aware a HERO10 bug on this.  So update your camera, and test again, share your findings (in a new thread).  Include the MISC/qrlog.txt for the time for the capture.
